uw-frame-components/portal/search/partials/search.html

```diff
@@ -7,7 +7,7 @@
             aria-label="Search bar enter the app you are looking for"
             focus-me="showSearchFocus"
             ng-show="$storage.typeaheadSearch"
-            typeahead="title as match.title for match in filterMatches | limitTo:10"
+            uib-typeahead="title as match.title for match in filterMatches | limitTo:10"
```

Comment:
Nope, according to [there latest doc](https://angular-ui.github.io/bootstrap/), all the attributes do not add in the `uib-` prefix